geps/gep-1619/index.md

```diff
@@ -363,71 +445,116 @@ type SessionPersistencePolicySpec struct {
     // +optional
     IdleTimeoutSeconds int64 `json:"idleTimeoutSeconds,omitempty"`
 
-    // SessionName defines the name of the persistent session token
-    // (e.g. a cookie name).
+    // Type defines the type of session persistence such as through
+    // the use a header or cookie. Defaults to cookie based session
+    // persistence.
     //
-    // Support: Extended
+    // Support: Core for "Cookie" type
+    //
+    // Support: Extended for "Header" type
```

Comment:
How about query parameter ( jsessionid is supported as both cookie and query param by java) ? Should we mention it here as extended ? Forgot about that...

Also not sure how to say that this should be a hint.

Reply:
I left it out because there isn't an active GWAPI implementation that exposes it, so it feels like it may never be used. Is it better to keep it minimal and add it only when we need? 

> Also not sure how to say that this should be a hint.

Sorry - what hint? Am I missing something?